packages/web-pkg/src/composables/actions/files/useFileActionsCreateNewFile.ts

```diff
@@ -208,27 +209,25 @@ export const useFileActionsCreateNewFile = ({
 
   const actions = computed((): FileAction[] => {
     const actions = []
-    for (const newFileHandler of unref(newFileHandlers) || []) {
-      const openAction = newFileHandler.action
+    for (const appFileExtension of unref(appNewFileMenuExtensions) || []) {
       actions.push({
         name: 'create-new-file',
         icon: 'add',
-        handler: (args) => handler(args, newFileHandler.ext, openAction),
-        label: () => newFileHandler.menuTitle($gettext),
+        handler: (args) => handler(args, appFileExtension.extension, appFileExtension),
+        label: () => appFileExtension.newFileMenu.menuTitle(),
         isEnabled: () => {
           return unref(currentFolder)?.canUpload({ user: userStore.user })
         },
         componentType: 'button',
         class: 'oc-files-actions-create-new-file',
-        ext: newFileHandler.ext
+        ext: appFileExtension.extension
       })
     }
     for (const mimeType of unref(mimetypesAllowedForCreation) || []) {
-      const openAction = false
       actions.push({
         name: 'create-new-file',
         icon: 'add',
-        handler: (args) => handler(args, mimeType.ext, openAction),
+        handler: (args) => handler(args, mimeType.ext, undefined),
```

Comment:
Why the undefined?

Reply:
As discussed, before it was `false`, so this is the same behaviour but correct typing.